### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HimashiRathnayake**
## Purpose
The purpose of this PR is to provide a guidance on  how to run the Authentication Flow Composer.

## Goals

- Provide a guidance on running the Authentication Flow Composer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
-# identity-authentication-flow-composer
-Authentication Flow Composer is a UI based tool to design and configure the authentication flow easily.
+# Authentication Flow Composer
+Authentication Flow Composer is a UI-based tool to design and configure the authentication flows for the WSO2 Identity
+Server. This approach allows users to define the authentication flows without any prior syntax knowledge.
+
+### Setup Identity Server for the application
```
Comment (brionmario) [inline README.md]:
```suggestion
### Configure Identity Server
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+       "PATCH",
+       "HEAD",
+       "DELETE",
+       "PATCH"
+    ]
+    exposed_headers = [ "Location" ]
+```    
+
+    
+* Start the Identity Server using following commands from `<IS_HOME>/bin`.
```
Comment (brionmario) [inline README.md]: Lets link to official docs for instructions.

https://is.docs.wso2.com/en/latest/setup/running-the-product/#starting-on-windowslinuxmac-os

Comment (HimashiRathnayake) [reply README.md]: Fixed with [5e702ed](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/11/commits/5e702ed5f05afdecdf92862e264eb12548626f48).

File: README.md (hunk 1 above)
Comment (HimashiRathnayake) [reply README.md]: Fixed with [5e702ed](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/11/commits/5e702ed5f05afdecdf92862e264eb12548626f48)

**Review by brionmario — APPROVED** [review]

**Pull request merged**